tools/kmod/sof_remove.sh

```diff
@@ -5,8 +5,17 @@
 
 TOPDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)
 
-# shellcheck source=case-lib/lib.sh
-source "$TOPDIR"/case-lib/lib.sh
+# Used only by more advanced error handling
+source_opt_libsh()
+{
+    local libsh="$TOPDIR"/case-lib/lib.sh
+    if test -e "$libsh"; then
+        # shellcheck source=case-lib/lib.sh
+        source "$libsh"
+    else
+        printf '%s warning: %s not found, not sourced\n' "$0" "$libsh"
```

Comment:
do we need to print this?

Reply:
Depends... would that be enough to get your approval? :-)

Reply:
> Depends... would that be enough to get your approval? :-)

yes ;)

Reply:
@ujfalusi done.